doc/supportedPlatformVersions.md

```diff
@@ -139,6 +141,7 @@
 - 7.0.100-preview.5.22307.18
 - 7.0.100-preview.6.22352.1
 - 7.0.100-preview.7.22377.5
+- 7.0.100-rc.1.22431.12
```

Comment:
If we are not going to release for stretch, we should create a constant that defines .NET 7's final stretch version. This constant will need to be used to update tests like [this](https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs#L571-L575) that use a stretch based image. 

See [this pull request](https://github.com/microsoft/Oryx/pull/1539) for an example of this being done for .NET 3.1 and .NET 6